autoload/lsp/ui/vim.vim

```diff
@@ -343,9 +343,13 @@ function! lsp#ui#vim#workspace_symbol() abort
         return
     endif
 
-    let l:query = inputdialog('query>', '', "\<ESC>")
-    if l:query ==# "\<ESC>"
-        return
+    if len(a:query) > 0
```

Comment:
```
if empty(a:query)
```

Reply:
I've made this change, but there's a lot of len() == 0 checks in the code. Is empty() new and using it is possibly a backwards-compatibility breaker? Github can't handle showing me blame for when it was added to vim/vim.

Reply:
empty() existed since 7.0034.